src/psyclone/psyir/nodes/psy_data_node.py

```diff
@@ -145,53 +145,8 @@ def __init__(self, ast=None, children=None, parent=None, options=None):
         self._psy_data_symbol_with_prefix = \
             self.add_psydata_class_prefix("psy_data")
 
-        # The use statement that will be inserted. Any use of a module
-        # of the same name that doesn't match this will result in a
-        # NotImplementedError at code-generation time.
-        self.use_stmt = "use {0}, only: "\
-            .format(self.add_psydata_class_prefix("psy_data_mod")) + \
-            ", ".join(self.add_psydata_class_prefix(symbol) for symbol in
-                      PSyDataNode.symbols)
-
-        # TODO #1185. Much of what is done in this constructor should either
-        # be done in a create() method or in the transformation that adds
-        # this node.
-
-        # A PSyData node always contains a Schedule
-        sched = self._insert_schedule(children)
-        super(PSyDataNode, self).__init__(ast=ast, children=[sched],
-                                          parent=parent)
-
-        # Get or create a symbol table so we can avoid name clashes
-        # when creating variables
-        if parent and self.ancestor(Routine):
-            symtab = self.ancestor(Routine).symbol_table
-        else:
-            # FIXME: This may not be a good solution
-            symtab = SymbolTable()
-
-        # Ensure that we have a container symbol for the API access
-        fortran_module = self.add_psydata_class_prefix(self.fortran_module)
-        try:
-            csym = symtab.lookup(fortran_module)
-        except KeyError:
-            csym = ContainerSymbol(fortran_module)
-            symtab.add(csym)
-
-        # The type symbol for PSyData variables
-        var_name = self.add_psydata_class_prefix("PSyDataType")
-        try:
-            type_sym = symtab.lookup(var_name)
-            if (not isinstance(type_sym.interface, GlobalInterface) or
-                    type_sym.interface.container_symbol is not csym):
-                raise InternalError(
-                    "Symbol table already contains a symbol named '{0}' but "
-                    "its interface does not refer to the '{1}' container.".
-                    format(var_name, csym.name))
-        except KeyError:
-            type_sym = TypeSymbol(var_name, DeferredType(),
-                                  interface=GlobalInterface(csym))
-            symtab.add(type_sym)
+        # The name of the PSyData variable that is used for this PSyDataNode
```

Comment:
This looks like a change that should already be on master. Please could you check.

Reply:
It was, but there was an import left behind that I deleted now.